bemserver_ui/templates/pages/services/cleanup/manage.html

```diff
@@ -36,18 +36,36 @@ <h5>Service state</h5>
         <div class="col">
             <div class="table-responsive-xl">
                 <table class="table table-sm table-hover table-bordered caption-top">
-                    <caption class="text-end">
-                        <small class="text-nowrap text-muted"><app-items-count first-item="1" last-item="{{ cleanup_timeseries | length }}" total-count="{{ cleanup_timeseries | length }}"></app-items-count></small>
-                    </caption>
+                    <div class="row mb-3">
+                        <div class="col">   
```

Comment:
Weird `<div class="col">` position. It is not supposed to be here...
This div should be inside `<caption>`, to display sort field on the left and items count on the right.

Reply:
Should look something like this:

`<caption class="d-flex justify-content-between align-items-center">
    <form id="formFiltersElmt" class="d-lg-flex d-grid align-items-center gap-2" action="{{ url_for(request.endpoint) }}" method="POST">
        ...
    </form>
    <small class="text-nowrap text-muted">
        <app-items-count first-item="1" last-item="{{ cleanup_timeseries | length }}" total-count="{{ cleanup_timeseries | length }}"></app-items-count>
    </small>
</caption>
<thead>
    ...`